python/dgl/data/chem/utils.py

```diff
@@ -141,7 +142,7 @@ def __call__(self, mol):
         return {self.atom_data_field: atom_features}
 
 
-def smile2graph(smile, add_self_loop=False, atom_featurizer=None, bond_featurizer=None):
+def smile2graph(smile, add_self_loop=False, atom_featurizer=DefaultAtomFeaturizer(), bond_featurizer=None):
```

Comment:
If you change the default arg, you need to also change the related documentation.

Reply:
Done